support-frontend/assets/components/footerCompliant/Footer.tsx

```diff
@@ -1,14 +1,13 @@
 // ----- Imports ----- //
-// @ts-expect-error - required for hooks
```

Comment:
I also fixed up the errors in this file as they were all pretty easy. There errors were:

- this comment
- using `Node` instead of `ReactNode`
- Not having a return type on the component